test/controllers/supplementary/ConsignmentReferencesControllerSpec.scala

```diff
@@ -85,12 +85,12 @@ class ConsignmentReferencesControllerSpec extends CustomExportsBaseSpec with Bef
     }
 
     "populate the form fields with data from cache" in {
-      val result = displayConsignmentReferencesPageTestScenario(
-        Some(ConsignmentReferences(ducr = Some(Ducr(exemplaryDucr)), lrn = "1234ABCD"))
-      )
+      val result = displayConsignmentReferencesPageTestScenario(Some(correctConsignmentReferences))
 
       contentAsString(result).replaceAll(" ", "") must include("name=\"ducr.ducr\"\nvalue=\"" + exemplaryDucr + "\"")
-      contentAsString(result).replaceAll(" ", "") must include("name=\"lrn\"\nvalue=\"1234ABCD\"")
+      contentAsString(result).replaceAll(" ", "") must include(
```

Comment:
If you have more than one `contentAsString` is better to extract it to variable

Reply:
Done